tests/tests_integration/test_api/test_simulators/seed/data.py

```diff
@@ -19,21 +21,35 @@
 
 random_str = random_string(10)
 
-resource_names = {
-    "simulator_external_id": "py_sdk_integration_tests",
-    "simulator_integration_external_id": "py_sdk_integration_tests_connector",
-    "simulator_model_external_id": f"py_sdk_integration_tests_model_{random_str}",
-    "simulator_model_revision_external_id": f"py_sdk_integration_tests_model_{random_str}_v1",
-    "simulator_model_file_external_id": "ShowerMixer_simulator_model_file_5",
-    "simulator_model_external_dependency_file_external_id": "ExtDependency_simulator_model_external_dependency_file",
-    "simulator_routine_external_id": f"pysdk_routine_{random_str}",
-    "simulator_test_data_set_id": None,
-    "simulator_test_data_set_external_id": data_set_external_id,
-}
+
+@dataclass
+class ResourceNames:
+    SIMULATOR_EXTERNAL_ID: str
+    SIMULATOR_INTEGRATION_EXTERNAL_ID: str
+    SIMULATOR_MODEL_EXTERNAL_ID: str
+    SIMULATOR_MODEL_REVISION_EXTERNAL_ID: str
+    SIMULATOR_MODEL_FILE_EXTERNAL_ID: str
+    SIMULATOR_ROUTINE_EXTERNAL_ID: str
+    SIMULATOR_TEST_DATA_SET_ID: str | None
+    SIMULATOR_TEST_DATA_SET_EXTERNAL_ID: str
+    SIMULATOR_MODEL_EXTERNAL_DEPENDENCY_FILE_EXTERNAL_ID: str
+
+
+resources = ResourceNames(
+    SIMULATOR_EXTERNAL_ID="py_sdk_integration_tests",
+    SIMULATOR_INTEGRATION_EXTERNAL_ID="py_sdk_integration_tests_connector",
+    SIMULATOR_MODEL_EXTERNAL_ID=f"py_sdk_integration_tests_model_{random_str}",
+    SIMULATOR_MODEL_REVISION_EXTERNAL_ID=f"py_sdk_integration_tests_model_{random_str}_v1",
+    SIMULATOR_MODEL_FILE_EXTERNAL_ID="ShowerMixer_simulator_model_file_5",
+    SIMULATOR_ROUTINE_EXTERNAL_ID=f"pysdk_routine_{random_str}",
+    SIMULATOR_TEST_DATA_SET_ID=None,
+    SIMULATOR_TEST_DATA_SET_EXTERNAL_ID=data_set_external_id,
+    SIMULATOR_MODEL_EXTERNAL_DEPENDENCY_FILE_EXTERNAL_ID="ExtDependency_simulator_model_external_dependency_file",
+)
 
 simulator = {
```

Comment:
module-level constants should be uppercased.
```suggestion
SIMULATOR = {
```